person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(_id, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly remove `_id` from the parameter list as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Constructor%20with%20name%2C%20age%2C%20and%20parent_permission%20as%20parameter.%20name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true.). The constructor with name, age, and parent_permission as parameter only.